svg/historical.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<title>Historical SVG features must be removed</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var removedInterfaces = [
+  "SVGAnimatedPathData",
+  "SVGColor",
+  "SVGDocument",
+  "SVGElementInstanceList",
+  "SVGExternalResourcesRequired",
+  "SVGFilterPrimitiveStandardAttributes",
```

Comment:
This should be in `mixinInterfaces`: https://drafts.fxtf.org/filters/#InterfaceSVGFilterPrimitiveStandardAttributes